src/renderer/shared/ui-entities/WalletManagement/WalletManagement.tsx

```diff
@@ -2,16 +2,26 @@ import { type PropsWithChildren, type ReactNode } from 'react';
 
 import { type Wallet } from '@/shared/core';
 import { cnTw } from '@/shared/lib/utils';
-import { BodyText, FootnoteText, Icon } from '@/shared/ui';
+import { BodyText, FootnoteText, Icon, Identicon } from '@/shared/ui';
 
 type Props = {
   wallet: Wallet;
   description?: string | ReactNode;
   meta?: ReactNode;
+  accountId: string | undefined;
```

Comment:
```suggestion
  accountId?: string;
```
And we can use `Address` or `AccountId` instead of meaningless `string`
We have a bunch of named types in `@/shared/core`

Reply:
I set it with `| undefined` to force devs to use the param anyway